l3xz_mapping/Waypoint waypoint
---
bool accepted
